partner_industry_parent/models/res_partner.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import SUPERUSER_ID, _, api, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def _search(
+        self,
+        args,
+        offset=0,
+        limit=None,
+        order=None,
+        count=False,
+        access_rights_uid=None,
+    ):
+        if self._context.get("search_default_industry_id"):
```

Comment:
`_context` is deprecated. Use `self.env.context.get()` instead.

Reply:
All right, fixed :-)